README.md

```diff
@@ -269,12 +410,13 @@ import { updateLocale } from '@americanexpress/one-app-ducks';
 // ...
 
 store.dispatch(updateLocale('be-BY'));
-
 ```
 
-#### `getLocalePack`
+##### `getLocalePack`
 
-Similar to `updateLocale`, but will attempt to load the locale of the requested country that is closest to the current active locale.
+Similar to `updateLocale`, but will attempt to load the locale of the requested country that is
+closest to the current active locale. This is used directly by One App, and it is unlikely that any
```

Comment:
In the `browser` API you use the `This is for internal use only` note to annotate that something should only be used by One App. Can we keep these consistent?

I do like this one better as it is more clear as to what you mean.